README.md

```diff
@@ -340,55 +349,55 @@ information, see about_Execution_Policies at http://go.microsoft.com/fwlink/?Lin
+- When installing the module on Windows you might run into an issue regarding long file names (LFN) due to the long paths of the generated schema files. If you install your module on a Linux master, and then use plugin sync you will likely not see this issue. If you are attempting to install the module on a Windows machine using `puppet module install puppetlabs-dsc` you may run into an error that looks similar to the following:
 
-~~~
+  ~~~
 Error: No such file or directory @ rb_sysopen - C:/ProgramData/PuppetLabs/puppet/cache/puppet-module/cache/tmp-unpacker20150713-...mof
 Error: Try 'puppet help module install' for usage
-~~~
+  ~~~
 
-For Puppet 4.2.2+ (and 3.8.2) we've lessened the possibility of the issue occurring based on the fixes in [PUP-4854](https://tickets.puppetlabs.com/browse/PUP-4854). However, a complete fix will become available in a version of Puppet that incorporates [PUP-4866](https://tickets.puppetlabs.com/browse/PUP-4866).
+For Puppet 4.2.2+ (and 3.8.2) we've decreased the possibility of the issue occurring based on the fixes in [PUP-4854](https://tickets.puppetlabs.com/browse/PUP-4854). A complete fix will become available in a future version of Puppet that incorporates [PUP-4866](https://tickets.puppetlabs.com/browse/PUP-4866).
+If you are affected by this issue, you can work around this by downloading the `.tar.gz` from the Forge and use `puppet module install` using the file rather than directly from the Forge.
 
-If you are affected by this issue, a known workaround is to download the `.tar.gz` from the forge and use `puppet module install` using the file rather than directly from the forge.
 
-- Windows 2003 is not supported.  Further, when installing the module on a Puppet master to the default `production` environment, this module ***will*** cause pluginsync to fail on Windows 2003 agents due to an issue with [LFN (long file names)](https://tickets.puppetlabs.com/browse/PUP-4866).  To workaround this, use [Puppet environments](https://docs.puppetlabs.com/puppet/latest/reference/environments.html) to create an environment separate from `production`, without the DSC module installed, to host Windows 2003 nodes.
+- Windows 2003 is not supported. Further, when installing the module on a Puppet master to the default `production` environment, this module ***will*** cause pluginsync to fail on Windows 2003 agents due to an issue with [LFN (long file names)](https://tickets.puppetlabs.com/browse/PUP-4866).  To work around this, use [Puppet environments](https://docs.puppetlabs.com/puppet/latest/reference/environments.html) to create an environment separate from `production`, without the DSC module installed, to host Windows 2003 nodes.
```

Comment:
This issue is quite a big one - we may want to somehow highlight it. It means that even if the customer DOES NOT want to use the dsc module with Windows 2003, ***the sheer presence of this module on the master*** will break the Windows 2003 agents - because all modules in an environment are synced to all agents. 

This is a HUGE problem and should probably be shown a little better to account for that.